sdk/communication/Azure.Communication.Chat/tests/SessionRecords/ChatClientsTest/ReadReceiptGS.json

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
Did recordings for threads and messages stay unchanged? Can we regen them too?

Reply:
I kind of refactored everything. The new recorded files are :
ThreadCGUD_MemberAUR_MessageGSU_NotificationT_AsyncAsync.json
ThreadCGUD_MemberAUR_MessageGSU_NotificationT.json
ReadReceiptGSAsyncAsync.json
ReadReceiptGS.json

The others got removed